aegis_control/launch/start_drivers.launch.py

```diff
@@ -68,6 +68,19 @@ def launch_setup(context: LaunchContext) -> list[LaunchDescriptionEntity]:
         launch_arguments=launch_args.items(),
     )
 
+    tool_cameras_driver = IncludeLaunchDescription(
```

Comment:
Let's improve the name notation:
```suggestion
    tool_rgb_cameras_driver = IncludeLaunchDescription(
```

Reply:
Done